src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -1129,8 +1152,15 @@ internal void OnOrganelleToPlaceSelected(string organelle)
         GD.Print("Editor action is now: " + editor.ActiveActionName);
     }
 
+    internal void OnCancelMoveClicked()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+        editor.CancelOrganelleMove();
```

Comment:
There should be a general method added to the microbe editor to cancel the currently active action